pkg/services/pluginsintegration/clientmiddleware/utils.go

```diff
@@ -4,6 +4,17 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 )
 
+const (
+	statusOK        = "ok"
+	statusError     = "error"
+	statusCancelled = "cancelled"
+
+	endpointCallResource   = "callResource"
+	endpointCheckHealth    = "checkHealth"
+	endpointCollectMetrics = "collectMetrics"
+	endpointQueryData      = "queryData"
+)
```

Comment:
Those are now shared between:

- `logger_middleware.go`
- `instrumentation_middleware.go`

Not sure if there's a better place for them, or maybe we should just repeat them?